patch/llpcPatch.cpp

```diff
@@ -308,6 +308,7 @@ void Patch::AddOptimizationPasses(
         passMgr.add(createSimpleLoopUnrollPass(optLevel));
         passMgr.add(CreatePatchPeepholeOpt());
         passMgr.add(createScalarizerPass());
+        passMgr.add(CreatePatchLoadScalarizer());
```

Comment:
This should also be added to the "cl::UseLlvmOpt == true" case around line 364. Or if we are not maintaining that case it should be removed.

Reply:
I agree we need add it to cl::UseLlvmOpt == true